### PR TITLE
Disable concurrency groups for clippy/coverage

### PR DESCRIPTION
**Pull request by franziskuskiefer**
I suspect that github.ref is from main for pull_request_target jobs such that these jobs get cancelled all the time.

### CODE REVIEW

**Review by kkohbrok — APPROVED** [review]
Sure, let's see if this works.

**Review by duesee — APPROVED** [review]
I wonder if we can not use a more specific ref instead of removing the cancelation completely.

**Comment by franziskuskiefer** [comment]
pushed this straight to main without waiting for (broken) CI to unblock the other PRs.

**Pull request merged**
